src/app/master-view/master-view.component.ts

```diff
@@ -1,24 +1,54 @@
 import { Component, OnInit } from '@angular/core';
-import { Northwind_JasonService } from '../services/northwind_jason.service';
+import { BehaviorSubject } from 'rxjs';
+import { ICustomer, Northwind_JasonService } from '../services/northwind_jason.service';
 
 @Component({
   selector: 'app-master-view',
   templateUrl: './master-view.component.html',
   styleUrls: ['./master-view.component.scss']
 })
 export class MasterViewComponent implements OnInit {
-  public northwindJasonApiCustomers: any = null;
+  public northwindJasonApiCustomers: ICustomer[] | null = null;
```

Comment:
I don't know whether this is more an issue with the data source. A better datasource will not have these issues?

Reply:
The issue is codegen cutting corners :D
@damyanpetev take note that `any` makes as awful dev experience, writting the types was faster for me than try to guess at each step.
This was mostly for writing bindings in the template, in the case of data consumed solely by the Grid it didn't make any difference, so I left it as `any`.